docs/dev/compiler-toolchain/overview.md

```diff
@@ -0,0 +1,66 @@
+# Overview
+
+This documentation introduces an LLVM-based compiler toolchain for smart contract languages with Ethereum Virtual Machine
+(EVM) support. The toolchain works on top of existing compilers and requires their output, which typically includes
+intermediate representations (IRs), abstract syntax trees (ASTs), and auxiliary contract metadata and documentation.
+At the moment, languages such as Solidity and Vyper are supported in this way.
+
+The toolchain roughly consists of the following parts:
+
+1. High-level source code compilers (`solc` and `vyper`)
+2. IR compilers, or LLVM front-ends (`zksolc` and `zkvyper`)
+3. [The LLVM framework](./llvm.md) with a zkEVM back-end, emitting the zkEVM text assembly
+4. The assembler, producing the zkEVM bytecode from the text assembly
+5. Hardhat plugins, setting up the environment
```

Comment:
Sure.
Let's remove parenthesis and code quoting in p.1-2, and add the links.
Since we've changed p.3 to `which emits`, I suggest changing p.4 to `which produces` and p.5 to `which sets up`.

Reply:
done